pkgs/development/python-modules/pipenv/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, virtualenv-clone
+, certifi
+, pytestCheckHook
+, requests
+, pytest-pypi
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "pipenv";
+  version = "2022.7.24";
+
+  src = fetchFromGitHub {
+    owner = "pypa";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-vxJ6AcBGM2+s97h8fbYHuQ4EesdZdDcrKgGljIuUG2A=";
+  };
+
+  buildInputs = [
+    certifi
+    virtualenv-clone
+  ];
+
+  # Fails with
+  # ImportError: cannot import name 'prepare_fixtures' from 'pytest_pypi.app'
```

Comment:
I suspect you're having no luck because it looks like pipenv use their own bundled copy of pytest-pypi (presumably because "upstream" pytest-pypi appears to be abandoned long ago)